app-utils/sensible-utils/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=sensible-utils
+PKGSEC=utils
+PKGDES="Utilities for sensible alternative selection"
```

Comment:
Utilities for configuring common utility alternatives

Reply:
Resolved.